src/main/java/uk/gov/laa/ccms/caab/bean/ActiveCase.java

```diff
@@ -12,6 +12,8 @@ public class ActiveCase {
   private String client;
   private String clientReferenceNumber;
 
+  private Integer applicationId;
```

Comment:
Is this not stored in the session already?

Reply:
It is indeed. Guess I was just reducing the number of params needed on a controller method.